tests/system/feature_store/test_spark_engine.py

```diff
@@ -271,13 +271,17 @@ def test_name():
     def test_output_subdir_path(self, url=True):
         return f"{self.output_dir(url=url)}/{self.test_name()}"
 
-    def set_targets(self, feature_set, also_in_remote=False):
+    def set_targets(self, feature_set, also_in_remote=False, explicit=False):
         dir_name = self.test_name()
         if self.run_local or also_in_remote:
             target_path = f"{self.output_dir(url=False)}/{dir_name}"
             feature_set.set_targets(
                 [ParquetTarget(path=target_path)], with_defaults=False
             )
+        elif explicit:
+            feature_set.set_targets(
+                [ParquetTarget(), NoSqlTarget()], with_defaults=False
```

Comment:
Why you need also the ParquetTarget if you are set explicit= passthrough?
And another question, You  don't think it is better to change the test it self, and to test if no target is created when using passthrough?